### ISSUE TITLE
Definition of 'perimeter' should be removed

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag2ict#423]
### Currently

'perimeter' is included in [Glossary Items with Specific Guidance](https://www.w3.org/TR/wcag2ict-22/#glossary-items-with-specific-guidance)

However, 'perimeter' is only used in SC 2.4.13 Focus Appearance, which is level AAA and not included in WCAG2ICT.

### Proposed

Remove 'perimeter' wherever it is currently mentioned in WCAG2ICT.

Include 'perimeter' in [Glossary Items Used only in AAA Success Criteria](https://www.w3.org/TR/wcag2ict-22/#glossary-items-used-only-in-aaa-success-criteria).

**Comment by maryjom** [comment]
I think at some point there was a cross reference between definitions (or the changes to the SCs having to do with Focus Appearance moving back to AAA) which caused this to be pulled in when we initially worked on definitions. I agree this term should moved as you propose.

**Comment by maryjom** [comment]
PR #439 makes the requested modification to the glossary terms and also updates the editor's notes and Comparison with the 2013 WCAG2ICT section to indicate "perimeter" was added, but to the section of terms used only in Level AAA.

**Comment by maryjom** [comment]
[TF resolution](https://www.w3.org/2024/07/11-wcag2ict-minutes#r04) to incorporate this change was made on 11 July. Closed as completed by PR #439.
